src/bun.js/webcore/response.zig

```diff
@@ -150,7 +150,7 @@ pub const Response = struct {
         globalThis: *JSC.JSGlobalObject,
     ) callconv(.C) JSC.JSValue {
         // https://developer.mozilla.org/en-US/docs/Web/API/Response/url
-        return ZigString.init(this.status_text).withEncoding().toValueGC(globalThis);
+        return ZigString.init(this.body.init.status_text).withEncoding().toValueGC(globalThis);
```

Comment:
There are other usages of `status_text` in the code. Those will need to be updated as well. A test that checks status_text is sent in the HTTP response from the server would be best.

Reply:
i've added tests in various places to ensure. fetch write is still WIP.